lib/inspec/resources/opa_cli.rb

```diff
@@ -0,0 +1,42 @@
+require "inspec/resources/opa"
+
+module Inspec::Resources
+  class OpaCli < Opa
+    name "opa_cli"
+    supports platform: "unix"
+    supports platform: "windows"
+
+    attr_reader :allow
+
+    def initialize(opts = {})
+      @policy = opts[:policy] || nil
+      @data = opts[:data] || nil
+      @query = opts[:query] || nil
+      fail_resource "policy and data are the mandatory for executing OPA." if @policy.nil? && @data.nil?
```

Comment:
"policy and data are mandataory..." - no "the".

Also, this should probably be a `||`.

Reply:
Isn't query mandatory as well?